### PR TITLE
Update RELEASE.md instructions to fit current setup

### LINKED ISSUE #64: RELEASE.md outdated
The instructions in RELEASE.md had become outdated after #41.

Until now we have used `bump2version` which is still documented in RELEASE.md. We have some projects being setup with bump2version and some setup with the jupyterhub/jupyterhub like system. I prefer bump2version in general once its setup and consistency, but I don't think its so important.

Should we revert back to use bump2version, or follow through the changes in #41 and update RELEASE.md to follow that pattern? I'll actually open two PRs because I can do it so quickly, merge one after comparing @jgwerner =D

```python
# from ltiauthenticator/__init__.py, this was removed
__version__ = '1.0.1.dev'



# _version.py was added with this
"""ltiauthenticator version info"""

version_info = (
    1,
    0,
    1,
    "dev",
)
__version__ = ".".join(map(str, version_info[:4]))

if len(version_info) > 4:
    __version__ = "%s%s" % (__version__, version_info[4])



# in setup.py, the following was added
# setup logic from github.com/jupyterhub/jupyterhub
v = sys.version_info
if v[:2] < (3, 6):
    error = "ERROR: LTIAuthenticator requires Python version 3.6 or above."
    print(error, file=sys.stderr)
    sys.exit(1)

# Get the current package version.
here = os.path.abspath(os.path.dirname(__file__))
version_ns = {}
with open(os.path.join("_version.py")) as f:
    exec(f.read(), {}, version_ns)



# in setup.py, the following was removed for a dynamic value
setup(
    name='jupyterhub-ltiauthenticator',
    version='1.0.1.dev',
```

### PR DESCRIPTION
**Pull request by consideRatio**
Fixes #64 by following through #41 to use the another release system by updating the docs in RELEASE.md and fixing a hardcoded string.

Another option is to merge #66. Closes #66.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -31,17 +31,45 @@ For you to follow along according to these instructions, you need:
...
+   patch version and a `dev` element, then make a commit.
 
-   ```bash
-   bump2version --no-tag patch
+   ```
+   git add _version.py
+   git commit -m "back to dev"
+   ```
+
+1. Push your two commits to master.
```
Comment (jgwerner) [inline RELEASE.md:52]: Change to `...commits to main`. There are other references to `master` so will ignore the rest.

**Review by jgwerner — CHANGES_REQUESTED** [review]
Just one minor comment (main vs master reference).

**Comment by consideRatio** [comment]
Updated @jgwerner! Before you merge though, please clarify you understand this is one of two different and exclusive approaches to close #64.

**Review by jgwerner — APPROVED** [review]
LGTM

**Comment by jgwerner** [comment]
@consideRatio yes it's clear that you can either merge this PR or #64 - I think both are sensible approaches and will let you decide which one to merge. Thanks!

**Comment by consideRatio** [comment]
Thanks @jgwerner! I merged and cut version 1.1.0 now.

**Pull request closed without merging**